packages/core/src/checkout-buttons/checkout-button-options.ts

```diff
@@ -70,6 +70,12 @@ export interface BaseCheckoutButtonInitializeOptions extends CheckoutButtonOptio
      */
     googlepayadyenv3?: GooglePayButtonInitializeOptions;
 
+    /**
+     * The options that are required to facilitate CybersourceV2 GooglePay. They can be
```

Comment:
CybersourceV2? I understand this integration is based on CybersourceV2's and we might have been end it up copy & paste big pieces of code and docs. There are 50 matches for this wording, please make sure to rename (we applies).